### PR TITLE
DDD plugin 101 with basic example

### PR DESCRIPTION
**Pull request by StephanPirnbaum**

### CODE REVIEW

File: src/main/tutorials/ddd-plugin/index.adoc [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+The Maven build can be triggered as usual on the command line:
+
+----
+mvn clean install
+----
+
+[[PreDefinedRules]]
+== Pre-Defined Rules
+
+The setup above activates the pre-defined group `java-ddd:Default that provides some basic constraints for
```
Comment (DirkMahler) [inline src/main/tutorials/ddd-plugin/index.adoc]: ` missing after the groupname

File: src/main/tutorials/ddd-plugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <java.version>1.8</java.version>
+    </properties>
+
+    <dependencies>
+        <!-- tag::dependency[] -->
+        <dependency>
+            <groupId>org.jqassistant.contrib.plugin</groupId>
+            <artifactId>jqassistant-java-ddd-plugin</artifactId>
+            <version>1.6.0</version>
```
Comment (DirkMahler) [inline src/main/tutorials/ddd-plugin/pom.xml:25]: scope provided

File: src/main/tutorials/ddd-plugin/pom.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        </dependency>
+        <!-- end::dependency[] -->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.8</version>
+        </dependency>
```
Comment (DirkMahler) [inline src/main/tutorials/ddd-plugin/pom.xml:34]: scope provided

**Review by DirkMahler — CHANGES_REQUESTED** [review]

File: src/main/tutorials/ddd-plugin/index.adoc (hunk 1 above)
Comment (StephanPirnbaum) [reply src/main/tutorials/ddd-plugin/index.adoc]: Ups, done

File: src/main/tutorials/ddd-plugin/pom.xml (hunk 2 above)
Comment (StephanPirnbaum) [reply src/main/tutorials/ddd-plugin/pom.xml:25]: Done

File: src/main/tutorials/ddd-plugin/pom.xml (hunk 3 above)
Comment (StephanPirnbaum) [reply src/main/tutorials/ddd-plugin/pom.xml:34]: Done

**Pull request merged**